### DISCUSSION TITLE
А вот бы ее научить на европейские или азиатские серверы PUBG коннектиться

### DISCUSSION THREAD

**Discussion by nolemociy** [Flowseal/zapret-discord-youtube#569] [Ideas]
Игра Playerunknown's Battleground коннектится только к тем серверам, к которым хочет сам.
Из России игра подключается к РУ-серверам. 

Есть такая штука - GearUP booster, позволяет играть на европейских серверах. Причем это не VPN, вроде как. Просто на сетевом уровне происходит подмена адресов серверов, к которым происходит коннект.

Можно ли научить эту софтинку такому же поведению, чтобы можно было подключаться к европейским, американским или азиатским серверам?

**Comment by Flowseal** [comment]
Как это вообще относится?
